### ISSUE TITLE
The number of allowed difficulty ownership change should be limited

### ISSUE THREAD

**Issue by Nivalyx** [ppy/osu-web#11443]
Related to [this recent incident](http://osu.ppy.sh/beatmapsets/2233033/discussion/-/events) where someone maliciously tried to "mass ping" a bunch of mappers/users by changing the ownership of a difficulty over and over.

![media-url](https://github.com/user-attachments/assets/59d5d7a1-066c-47c1-8947-d5668ee85e59)

I feel like \*really\* there should be a limit as to how much regular users could change the difficulty ownership on a set they own so that this feature don't get abused again in the future... Somewhere around 3x changes per diff per day maybe?

**Comment by nanaya** [comment]
Yeah I don't know when it's also a fast track to getting yourself banned.
